src/model/user.rs

```diff
@@ -166,7 +166,19 @@ impl CurrentUser {
     /// ```
     #[inline]
```

Comment:
```suggestion

```
This should no longer be marked as `#[inline]`. I fear it would cause more harm than good.
It's a beneficial attribute for small functions, as it permits them to be inlined across crate boundaries. But if they're longer than three statements, then I believe it shouldn't apply to them anymore.

Reply:
Fixed